### PR TITLE
docs: add particle network to signers

### PR DESCRIPTION
**Pull request by TABASCOatw**
Adds the following:
- Page under 'signers' for Particle Network
- Corresponding code snippet under 'snippets'
- Altered sidebar to account for new page

---

### Detailed summary

- Added a new page for Particle Network in the site's documentation
- Imported necessary modules for Particle Network integration
- Configured ParticleNetwork with project ID, client key, and app ID
- Created a SmartAccountSigner using ParticleProvider and WalletClientSigner
- Added code snippets for using Particle Network with LightSmartContractAccount

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: site/snippets/particle.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { ParticleNetwork } from '@particle-network/auth';
+import { ParticleProvider } from '@particle-network/provider';
+import { createWalletClient, custom } from "viem";
+import { WalletClientSigner, SmartAccountSigner } from "@alchemy/aa-core";
+
+const particle = new ParticleNetwork({
+  projectId: process.env.REACT_APP_PROJECT_ID as string,
+  clientKey: process.env.REACT_APP_CLIENT_KEY as string,
+  appId: process.env.REACT_APP_APP_ID as string,
+  chainName: 'polygon',
```
Comment (denniswon) [inline site/snippets/particle.ts:10]: Can you add some notion about choosing the right FACTORY_CONTRACT_ADDRESS, ENTRY_POINT_CONTRACT_ADDRESS for specific chain specified here in ParticleNetwork signer?

Comment (TABASCOatw) [reply site/snippets/particle.ts:10]: Sure, done

**Review by moldy530 — DISMISSED** [review]

**Comment by avasisht23** [comment]
Hey the `site/snippets/particle.ts` file has lint errors. can you run prettier @TABASCOatw

**Comment by TABASCOatw** [comment]
> Hey the `site/snippets/particle.ts` file has lint errors. can you run prettier @TABASCOatw

Sure. How's this?

**Comment by TABASCOatw** [comment]
Found the issues- should be good now @avasisht23

**Comment by avasisht23** [comment]
hmm @TABASCOatw still failing: https://github.com/alchemyplatform/aa-sdk/actions/runs/6635020413/job/18025454530?pr=152. how are you running your lint/prettier?

**Comment by TABASCOatw** [comment]
> hmm @TABASCOatw still failing: https://github.com/alchemyplatform/aa-sdk/actions/runs/6635020413/job/18025454530?pr=152. how are you running your lint/prettier?

Just gave it another try but with prettier. Tried manual prettier first, then eslint to fix detected issues, now just ran it through standard cli prettier.

**Comment by avasisht23** [comment]
hmm I'm not sure what's going on with the linter, but I'm just going to make a branch with these changes and then mark you as a collaborator and reviewer.

closing in favor of #194

**Pull request closed without merging**